flake.nix

```diff
@@ -48,7 +48,9 @@
         };
 
         checks = lib.filterAttrs (k: v: v != null)
-          (exposed.checks // exposed-stable.checks);
+          (exposed.checks
+          // (lib.mapAttrs' (name: value: lib.nameValuePair "stable-${name}" value)
+            exposed-stable.checks));
```

Comment:
These two sets had the same names, so only the checks from the second set (`exposed-stable`) were being built. This led to a confusing situation where `nix flake check` would fail but building the corresponding package with `nix build .#opam` would succeed.